src/manager.cpp

```
@@ -749,6 +753,9 @@ int boot(const po::variables_map& vm) {
 
     auto module_handles =
         start_modules(*config, mqtt_abstraction, ignored_modules, standalone_modules, ms, status_fifo, retain_topics);
+    if (module_handles.empty()) {
+        return EXIT_FAILURE;
```

Should this be considered a failure?

---

That's certainly debatable. The current behavior is that everest will happily load empty configs, but it doesn't signal a completed startup via the status fifo or logs. So for a user it just "hangs" without doing anything else. I thought failing early would be better in this case, but we can also leave the current behavior in place if that's more desirable